cmd/license-register.go

```diff
@@ -170,12 +170,20 @@ func mainLicenseRegister(ctx *cli.Context) error {
 	regInfo := getClusterRegInfo(getAdminInfo(aliasedURL), clusterName)
 
 	alreadyRegistered := false
-	apiKey, _, e := getSubnetCreds(alias)
-	fatalIf(probe.NewError(e), "Error in fetching subnet API Key")
-	if len(apiKey) > 0 {
-		alreadyRegistered = true
-		if len(accAPIKey) == 0 {
-			accAPIKey = apiKey
+	if len(accAPIKey) == 0 {
+		apiKey, _, e := getSubnetCreds(alias)
+		fatalIf(probe.NewError(e), "Error in fetching subnet API Key")
+		if len(apiKey) > 0 {
+			alreadyRegistered = true
+			if len(accAPIKey) == 0 {
+				accAPIKey = apiKey
+			}
+		}
+	} else {
+		apiKey := getSubnetAPIKeyFromConfig(alias)
+		lic := getSubnetLicenseFromConfig(alias)
+		if len(apiKey) > 0 || len(lic) > 0 {
+			alreadyRegistered = true
```

Comment:
```suggestion
		if len(getSubnetAPIKeyFromConfig(alias)) > 0 {
			alreadyRegistered = true
```
Now api key is the main indicator of registration. So we need not look for license when checking if the cluster is already registered.

Reply:
done, thanks